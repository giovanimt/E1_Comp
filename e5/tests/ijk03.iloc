loadI 923 => r0 
addI rbss, 8 => r1 
store r0 => r1 
