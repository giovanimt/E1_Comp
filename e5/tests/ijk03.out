00009519 923
